src/app/routes/edges/routes/PrivateEdgeList/index.js

```diff
@@ -1252,7 +1252,7 @@ class PrivateEdgeList extends React.Component {
     const hasWriteAccessInCluster = (projectName) => {
       let hasWriteAccess = false;
       var allPermissions = this.props.userAndRoleDetail.spec.permissions
-        .filter((obj) => obj.project === projectName)
+        .filter((obj) => obj.project === projectName || obj.role === "ADMIN")
```

Comment:
check for a project role or org scoped role which is applicable to all projects.

Reply:
Added project role / org scoped role validation